Source/Core/Core/DSP/LabelMap.cpp

```diff
@@ -42,16 +42,23 @@ void LabelMap::RegisterDefaults()
   }
 }
 
-void LabelMap::RegisterLabel(std::string label, u16 lval, LabelType type)
+bool LabelMap::RegisterLabel(std::string label, u16 lval, LabelType type)
 {
   const std::optional<u16> old_value = GetLabelValue(label);
-  if (old_value && old_value != lval)
+  if (old_value)
   {
-    WARN_LOG_FMT(AUDIO, "Redefined label {} to {:04x} - old value {:04x}\n", label, lval,
-                 *old_value);
-    DeleteLabel(label);
```

Comment:
I'm not familiar with this code, this isn't needed anymore because we early out and don't add the new label?

Reply:
Yeah, I changed the behavior from replacing the old label with a new one to not replacing the label.

I also think the behavior was buggy if the old label value matched the new one, as then there would be 2 labels in the list with the same value.

DSPSpy has an option to continue assembling even if errors occur, so we do need to choose a behavior.